aptos-move/aptos-release-builder/src/components/framework.rs

```diff
@@ -48,7 +48,7 @@ pub fn generate_upgrade_proposals(
         repository.checkout_tree(commit.as_object(), None)?;
         commit_info
     } else {
-        git_version::git_version!().to_string()
```

Comment:
What's the diff here?

Reply:
The issue was that the `git_version` macro didn't work quite well with our docker build system. The new `commit_hash` will contain a plain hash instead of the tag+hash schema